poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java

```diff
@@ -100,14 +101,21 @@
    public static final String ContactClosureSensor = "ContactClosureSensor";
    public static final String SmokeSensor = "SmokeSensor";
    public static final String WaterSensor = "WaterSensor";
+   public static final String VibrationSensor = "Vibration";
+
+   public static final String Humidity = "Humidity";
```

Comment:
Why do we still need to define the metric name here?  I remember we have a Metric Name class already.

Reply:
1>These new constants are used in the method of updatePduMetricformular. We need to check the type of sensor asset, we will record the metric name by the type of sensors.
2>These constants just used to aviod hardcode in method of updatePduMetricformular.

Reply:
Removed